src/templates/activityTemplate/types.ts

```diff
@@ -0,0 +1,30 @@
+export interface ActivityPageData {
```

Comment:
These types are unnecessary, since they are created automatically by the graphql-codegen-plugin.

For example `Activity's` type can be found in:
```typescript
import { StrapiActivity } from 'src/graphql-types';
```